pkgs/top-level/all-packages.nix

```diff
@@ -34635,8 +34635,8 @@ with pkgs;
 
   qpdfview = libsForQt5.callPackage ../applications/office/qpdfview { };
 
-  qtile-unwrapped = callPackage ../applications/window-managers/qtile { };
-  qtile = callPackage ../applications/window-managers/qtile/wrapper.nix { };
+  qtile-unwrapped = python3.pkgs.qtile-unwrapped;
```

Comment:
```suggestion
```
please add an alias to aliases.nix which points qtile-unwrapped to python3.pkgs.qtile